requirements.txt

```diff
@@ -0,0 +1,55 @@
+antlr4-python3-runtime==4.7.2
```

Comment:
Can this be bought up to , e.g., 4.13.1 safely? I can see versions 4.8 thru 4.13.1 available via `conda`.